WordPress/src/main/java/org/wordpress/android/models/ReaderTagList.java

```diff
@@ -60,9 +63,9 @@ public ReaderTagList getDeletions(ReaderTagList tagList) {
             return deletions;
         }
 
-        for (ReaderTag thisTag: this) {
-            if (!tagList.hasSameTag(thisTag)) {
-                deletions.add(thisTag);
+        for (ReaderTag tag: this) {
```

Comment:
Thanks for catching that bug, @daniloercoli! I've fixed that in 5186703 and also renamed the variables for clarity.